lms/templates/edxnotes/toggle_notes.html

```diff
@@ -8,12 +8,13 @@
   edxnotes_visibility_url = reverse("edxnotes_visibility", kwargs={"course_id": course.id})
 %>
 <div class="wrapper-utility edx-notes-visibility">
+  <span class="action-toggle-message" aria-hidden="true">${_("Hiding Notes")}</span>
```

Comment:
@polesye and @jmclaus, this is a new bit of UI to help communicate what the system is doing. I need your help doing two things within the JS:

1. Toggling the visibility - this element is hidden and is only planned to show for a temporary amount of time. I have a CSS animation to handle that, but the class ``is-fleeting`` needs to be freshly added each time a user toggles the notes control on/off
2. The text for this element should show the correct state of the system, so when a user toggles off notes, "Hiding Notes" should be displayed, when a user toggles on notes, "Showing Notes" should be displayed in its place.

Can you folks help to wire that up?